Bio/SeqFeature.py

```diff
@@ -237,7 +245,7 @@ def _set_location_operator(self, value):
                                  doc="Location operator for compound locations (e.g. join).")
 
     def __repr__(self):
-        """A string representation of the record for debugging."""
+        """Represent the feature's sequence as a string for debugging."""
```

Comment:
No, it does not give the feature's sequence as a string (which might be ``ACGT...``). In the original ``record`` was misleading, would ``feature`` be better?